client/components/app.jsx

```diff
@@ -1,21 +1,43 @@
 import React from 'react';
 import WriteReview from './writeReview';
 import Reviews from './reviews';
+import helpers from '../ajaxHelpers/ajaxHelpers';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-
+      reviews: [],
+      restaurant: 'string',
+      restId: 'number',
```

Comment:
if you're expecting a number here I'd suggest having a number as a placeholder

Reply:
Herp derp. Thanks :)